domain/src/Commands/DomainCommands.php

```diff
@@ -0,0 +1,1108 @@
+        $entity->save();
+      }
+    }
+    return $count;
+  }
+
+  /**
+   * Reassign entities of the supplied type to the $policy domain.
+   *
+   *
+   *  $options = [
+   *    'entity_filter' => 'node',
+   *    'policy' => 'prompt' | 'default' | 'ignore'
+   *    'multidomain' => FALSE,
+   *  ];
+   *
+   * @param DomainInterface[] $domains
+   *   List of the domains to reassign content away from.
+   */
+  protected function reassignLinkedEntities($domains, array $options = []) {
+    $entity_typenames = $this->findDomainEnabledEntities();
+    $new_domain = NULL;
+    if ($options['policy']) {
+
+    }
+    foreach($entity_typenames as $name) {
+      if (empty($options['entity_filter']) || $options['entity_filter'] === $name) {
+        foreach($domains as $domain) {
+          $ids = $this->enumerateDomainEntities($name, $domain);
+          if (!empty($ids)) {
+            $this->reassignEntities($name, $domain, $new_domain);
+          }
+        }
+      }
+    }
+  }
+
+  // -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+  /**
+   * List active domains for the site.
+   *
+   * @option inactive
+   *   Show only the domains that are inactive/disabled.
+   * @option active
+   *   Show only the domains that are active/enabled.
+   * @usage drush domain:list
+   *   List active domains for the site.
+   * @usage drush domains
+   *   List active domains for the site.
+   *
+   * @command domain:list
+   * @aliases domains,domain-list
+   *
+   * @field-labels
+   *   weight: Weight
+   *   name: Name
+   *   hostname: Hostname
+   *   scheme: Scheme
+   *   status: Status
+   *   is_default: Default
+   *   domain_id: Domain Id
+   *   id: Machine name
+   * @default-fields domain_id,id,name,hostname,status,is_default
+   *
+   * @return \Consolidation\OutputFormatters\StructuredData\RowsOfFields
+   *
+   * @throws \Drupal\domain\Commands\DomainCommandException
+   */
+  public function listDomains(array $options = ['inactive' => null, 'active' => null]) {
+    $domain_storage = $this->getStorage();
+
+    // Load all domains:
+    $domains = $domain_storage->loadMultipleSorted(NULL);
+
+    if (empty($domains)) {
+      $this->logger()->warning(dt('No domains have been created. Use "drush domain:add" to create one.'));
+      return new RowsOfFields([]);
+    }
+
+    $keys = [
+      'weight',
+      'name',
+      'hostname',
+      'scheme',
+      'status',
+      'is_default',
+      'domain_id',
+      'id',
+    ];
+    $rows = [];
+    /** @var \Drupal\domain\DomainInterface $domain */
+    foreach ($domains as $domain) {
+      $row = [];
+      foreach ($keys as $key) {
+        $v = $domain->get($key);
+        switch($key) {
+          case 'status':
+            if (($options['inactive'] && $v) || ($options['active'] && !$v)) {
+              continue 3; // switch, for, for
+            }
+            $v = ($v) ? dt('Active') : dt('Inactive');
```

Comment:
I find this shorthand `$v = ($v)` hard to parse. Id rather use `empty($v) ? dt('Active') : dt('Inactive');`

Reply:
Done